test/ui-automation/test/specs/waci-flow.js

```diff
@@ -131,7 +149,7 @@ async function waciFlow(version, ctx) {
     await waciShareDemoBtn.waitForExist();
     await waciShareDemoBtn.click();
 
-    const vcName = await $("span*=Permanent Resident Card");
+    const vcName = await $("span*=Expired PR Card");
```

Comment:
Rather than modifying existing test case, can we add a new test ?

Something like below
1. keep all WACI flow as it is ie, validate without renaming
2. rename credential
3. share credential with update name along with name validation

Reply:
done